content/vue/en/screen.md

```diff
@@ -79,7 +79,7 @@ export default {
 </script>
 ```
 
-We also change the `App` component to render the `InboxScreen` (eventually we would use a router to choose the correct screen, but let's not worry about that here):
+We also change the `App` component to render the `InboxScreen` component (eventually we would use a router to choose the correct screen, but let's not worry about that here):
```

Comment:
I don't know if it's necessary to add this word because the name of the component itself indicates what it's referring to.

Reply:
Then it may be better to say `to render InboxScreen` instead of `to render the InboxScreen`. Otherwise, the sentence looks a bit strange to me (disclaimer: I'm not a native english speaker).

Reply:
`to render InboxScreen` makes sense.

Reply:
Actually, I found a lot of sentences to like that (`to render the X`) through many files, so I've reverted this change in order to avoid changing a lot of files.

Reply:
👍